modules/transactions/src/test/java/org/apache/ignite/internal/tx/TxManagerTest.java

```diff
@@ -116,6 +118,7 @@
  */
 @ExtendWith(ConfigurationExtension.class)
 @ExtendWith(ExecutorServiceExtension.class)
+@WithSystemProperty(key = IgniteSystemProperties.COLOCATION_FEATURE_FLAG, value = "true")
```

Comment:
Why is this needed? What about testing the non-colocated approach?

Reply:
>Why is this needed?

Because without this flag the assertion I added in this PR will trigger.

>What about testing the non-colocated approach?

I duplicated the test to test for both approaches